perses/annihilation/new_relative.py

```diff
@@ -70,6 +75,12 @@ def __init__(self, topology_proposal, current_positions, new_positions, use_disp
         self.softcore_alpha=0.5
         self.softcore_beta=12*unit.angstrom**2
```

Comment:
Hm, that decision happened in the previous PR, and I assumed this strategy (which is what was in the previous hybrid implementation as well) was ok since the PR was approved. We can discuss which is better, but I think a different PR is appropriate for changing it (if necessary).